test/unit/lib/intent-validator/content-transform.spec.ts

```diff
@@ -157,7 +157,7 @@ describe('Content transformation', function () {
                 type: 'contentType/transformEntries'
               }
             },
-            message: '"string" is not a valid type for the content transformation property "shouldPublish". Expected "boolean".',
+            message: '"undefined" is not a valid type for the content transformation property "shouldPublish". Expected "alternatives".',
```

Comment:
This new error message isn't really useful anymore for people.  
Maybe we could figure out in https://github.com/contentful/contentful-migration/blob/0203edf5267b07450ae0cb73e37660d5c456185f/src/lib/intent-validator/schema-validator.ts#L78 whether we are dealing with `Joi.alternatives` and figure out the different types that are allowed.  
Either that, or somehow change the message for the `alternatives` validation so that it doesn't say `Expected "alternatives"`.

Reply:
@TimBeyer error message fixed with tests